gridstatus/spp.py

```diff
@@ -782,55 +784,130 @@ def get_interconnection_queue(self, verbose=False):
 
         return queue
 
-    @lmp_config(
-        supports={
-            Markets.REAL_TIME_5_MIN: ["latest", "today", "historical"],
-            Markets.DAY_AHEAD_HOURLY: ["latest", "today", "historical"],
-        },
-    )
+    def get_lmp_real_time_5_min_by_location(
+        self,
+        date,
+        end=None,
+        location_type=LOCATION_TYPE_ALL,
```

Comment:
we don't need to change this for this PR since it's preexisting, but just an overall comment to keep in mind. 

 I think we should eventually move away from doing any `location_type` filtering in `gridstatus` in the LMP method. we should focus library on simply querying the ISOs API and returning cleaned data as they provide it. no benefit of maintaining this extra logic to filter down what the ISOs return.

we can do this type of filtering much better on the hosted API.

Reply:
I agree. We already fetch all the data, and we should return it all to the caller who can filter if necessary.